jujugui/static/gui/src/app/components/entity-details/header/_header.scss

```diff
@@ -31,11 +31,13 @@
     border-bottom: 1px solid $light-mid-grey;
     border-top-left-radius: $border-radius;
     animation: slideIn .5s;
+    pointer-events: none;
```

Comment:
Please add a comment explaining why this is required here so that we don't incur in the same regression again.